base/cvd/cuttlefish/host/commands/cvd/server_command/display.cpp

```diff
@@ -153,11 +153,10 @@ class CvdDisplayCommandHandler : public CvdServerHandler {
     const auto selector_args = cvd_common::ConvertToArgs(selector_opts.args());
 
     auto instance = CF_EXPECT(
-        instance_manager_.SelectInstance(selector_args, extra_queries, envs));
-    const auto& instance_group = instance.ParentGroup();
-    const auto& home = instance_group.HomeDir();
+        instance_manager_.SelectInstance(selector_args, envs, extra_queries));
+    const auto& home = instance.GroupInfo().home_dir;
```

Comment:
I don't think this issue occurs in this CL, buy FYI creating a `const auto&` reference like this would be safe with one intermediate level, but not two:

[Compiler explorer](https://godbolt.org/z/jjes18nGe)

`const` reference lifetime extension is still not intuitive for me, so thought it might be worth mentioning.

Reply:
ack